components/home/CurrentCycle.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+  TouchableOpacity, View,
+} from 'react-native';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+
+const NameText = styled.Text`
+   color: #EFEFEF;
+   font-size: 32px;
+   font-family: 'Montserrat_500Medium';
+`;
+
+const Title = styled.Text`
+  font-family: 'Montserrat_600SemiBold';
+  font-size: 24px;
+  margin: 0px 6%;
+  paddingBottom: 15px;
+`;
+
+const Subtext = styled.Text`
+   color: #EFEFEF;
+   font-size: 20px;
+   position: absolute;
+   bottom: 25px;
+   left: 25px;
+   font-family: 'Montserrat_500Medium';
+`;
+
+const CurrentCycle = (props) => {
+  const {
+    color, subtext, name, onPress,
+  } = props;
+
+  const StyledView = styled(TouchableOpacity)`
+      background-color: ${color};
+      width: 90%;
+      height: 210px;
+      border-radius: 20px;
+      padding: 15px 0px;
+      padding-right: 15px;
+      padding-left: 25px;
```

Comment:
nit: 
padding: 15px 15px 15px 25px; would do the same thing as the 3 padding statements you have.

Reply:
similarly, margin: 10px 10px 10px 20px;